backend/apps/nest/models/apikey.py

```diff
@@ -0,0 +1,66 @@
+"""Nest app API key model."""
+
+import hashlib
+import secrets
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class APIKey(models.Model):
+    """API key model."""
+
+    key_hash = models.CharField(max_length=64, unique=True, db_index=True)
+    key_suffix = models.CharField(max_length=4, blank=True)
+    name = models.CharField(max_length=100, null=False, blank=False)
+    revoked = models.BooleanField(default=False)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name="api_keys", db_index=True
+    )
+    created_at = models.DateTimeField(auto_now_add=True)
+    expires_at = models.DateTimeField(null=True, blank=True)
+
+    class Meta:
```

Comment:
You must keep the `Meta` place and DB table name consistent w/ other models. It has to be `nest_api_keys`